doc/Table.md

```diff
@@ -0,0 +1,281 @@
+| delayBound              | int(11) | An upper bound on the time (in seconds) between sending a result to a client and receiving a reply 
+| redundancy              | int(11) | The number of redundant calculations. Set to two or more ot achieve redundancy 
+| copies                  | int(11) | The number of copies of the workunit to issue to clients
+| errors                  | int(11) | The number of errors from clients before the workunit is declared to have an error 
+| numIssues               | int(11) | The total number of clients to issue the work unit to before it is declared to have an error 
+| resultsWithoutConcensus | int(11) | The total number of returned results without a concensus is found before the workunit is declared to have an error 
+| appName                 | text    | The app name (usually is sixtrack) 
+| appVer                  | int(11) | The app version 
+
+## env
+| Field          | Type    | Description
+|----------------|---------|------------
+| madx_exe       | text    | The madx executable
+| sixtrack_exe   | text    | The sixtrack executable 
+| study_path     | text    | The absolute path of this study 
+| preprocess_in  | text    | The input folder for preprocess jobs 
+| preprocess_out | text    | The output folder for preprocess jobs 
+| sixtrack_in    | text    | The input folder for sixtrack jobs 
+| sixtrack_out   | text    | The output folder for sixtrack jobs 
+| gather         | text    | The input folder for gather job which is used for collecting results 
+| templates      | text    | The template folder 
+| boinc_spool    | text    | The boinc spool for submitting jobs to boinc 
+| test_turn      | int(11) | When submitting jobs to boinc, we need to test the jobs on HTCondor firstly 
+| emit           | double  | Normalized emittance 
+| gamma          | double  | The gamma 
+| kmax           | int(11) | The number of phase space angles 
+| boinc_work     | text    | The absolute path of work directory 
+| boinc_results  | text    | The absolute path of results directory 
+| surv_percent   | int(11) | When testing jobs on HTCondor, survival percent decides whether to submit jobs to boinc
+
+## templates
+| Field     | Type | Description 
+|-----------|------|------------
+| mask_file | blob | The mask file of madx input file 
+| fort_3    | blob | The mother file of fort.3 
+
+## preprocess_wu
+| Field      | Type       | Description 
+|------------|------------|------
+| wu_id      | int(11)    | The work unit id
+| job_name   | text       | The job name  
+| input_file | blob       | The input file for preprocess.py  
+| batch_name | text       | The batch name of this job in HTCondor
+| unique_id  | text       | The unique id of this job on HTCondor, usually is ClusterId.ProcId  
+| status     | text       | The status of this job, 'complete', 'incomplete' or 'submitted'  
+| task_id    | int(11)    | The task id which point to a submission  
```

Comment:
I think that the description here should be something like: "id of the latest submitted task or of the successful/reference task"